chain/index/ddls.go

```diff
@@ -0,0 +1,82 @@
+package index
+
+import "database/sql"
+
+const DefaultDbFilename = "chainindex.db"
+
+var ddls = []string{
+	`CREATE TABLE IF NOT EXISTS tipset_message (
+		message_id INTEGER PRIMARY KEY,
```

Comment:
as discussed in zoom - this is poorly named, it should just be `id`

Reply:
@rvagg Have fixed this by changing the name to `id`.